src/main/java/Pages/App/IdTypeStep.java

```diff
@@ -0,0 +1,6 @@
+package Pages.App;
+
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.Steper.IdType.IdTypeStepPage;
+
+public class IdTypeStep extends IdTypeStepPage {
```

Comment:
Step class should not extend the page class.
Step is just a separate class that describes specific part of the page, so the instance of IdTypeStep should be created in constructor of IdTypeStepPage

same is valid for other step classes

Reply:
done